tsconfig.adf.json

```diff
@@ -24,6 +24,7 @@
       "@alfresco/adf-testing/shared": ["../alfresco-ng2-components/lib/testing/src/lib/shared"],
       "@alfresco/adf-core": ["../alfresco-ng2-components/lib/core"],
       "@alfresco/adf-core/*": ["../alfresco-ng2-components/lib/core/*/public-api.ts"],
+      "@alfresco/adf-core/shell": ["../alfresco-ng2-components/lib/core/shell/src/index.ts"],
```

Comment:
Why do you need the extra path alias definition? :O

Reply:
Needed for running the application with `adf` configuration.